src/main/java/com/mixam/mxjdf4/sdk/Component.java

```diff
@@ -90,4 +91,11 @@ public void setType(ComponentType type) {
         this.type = type;
     }
 
+    public int getProductionDays() {
```

Comment:
Can you move production days to article level please. We dont say its 7 days for the body and 3 days for the cover!